website/docs/r/firewall_section.html.markdown

```diff
@@ -85,7 +87,7 @@ The following arguments are supported:
   * `action` - (Required) Action enforced on the packets which matches the firewall rule. [Allowed values: "ALLOW", "DROP", "REJECT"]
   * `applied_to` - (Optional) List of object where rule will be enforced. The section level field overrides this one. Null will be treated as any. [Supported target types: "LogicalPort", "LogicalSwitch", "NSGroup"]
   * `destination` - (Optional) List of the destinations. Null will be treated as any. [Allowed target types: "IPSet", "LogicalPort", "LogicalSwitch", "NSGroup", "MACSet" (depending on the section type)]
-  * `destinations_excluded` - (Optional) Negation of the destination.
+  * `destinations_excluded` - (Optional) When this boolean flag is set to true, the rule destinations will be negated.
   * `direction` - (Optional) Rule direction in case of stateless firewall rules. This will only considered if section level parameter is set to stateless. Default to IN_OUT if not specified. [Allowed values: "IN", "OUT", "IN_OUT"]
   * `disabled` - (Optional) Flag to disable rule. Disabled will only be persisted but never provisioned/realized.
```

Comment:
just noticed - its really weird to name this "disabled" and not "enabled".. guess its too late now

Reply:
You are right, but it will be a bad idea to change this now.